pkgs/fujiapple852/trippy/registry.yaml

```diff
@@ -18,7 +18,7 @@ packages:
     replacements:
       amd64: x86_64
       darwin: apple-darwin
-      linux: unknown-linux-gnu
+      linux: unknown-linux-musl
```

Comment:
v0.7.0 doesn't have musl binary.

https://github.com/fujiapple852/trippy/releases/tag/0.7.0

The original setting is old, so we should re-scaffold.

Reply:
Thanks for the tip!